homeassistant/components/reddit/sensor.py

```diff
@@ -37,17 +42,20 @@
     vol.Required(CONF_USERNAME): cv.string,
     vol.Required(CONF_PASSWORD): cv.string,
     vol.Required(CONF_SUBREDDITS): vol.All(cv.ensure_list, [cv.string]),
+    vol.Optional(CONF_SORT_BY, default='hot'): cv.string,
```

Comment:
This requires additional [validation](https://developers.home-assistant.io/docs/en/development_validation.html#limit-the-values).